### PR TITLE
Always compile our bits with 'checked : true'

### PR DESCRIPTION
**Pull request by olkononenko**
I've run validation builds in BVT queues - there was no overflow. 

[AB#1534271](https://dev.azure.com/mseng/708e929f-6bd5-415a-8daf-25b1dac08dd8/_workitems/edit/1534271)

### CODE REVIEW

File: Public/Src/Engine/Dll/HistoricTableSizes.cs [hunk 1]
```diff
@@ -199,7 +199,7 @@ public override bool Equals(object obj)
         /// <inheritdoc/>
         public override int GetHashCode()
         {
-            return HashCodeHelper.Combine(Count, SizeInBytes);
+            return HashCodeHelper.Combine(Count, (int)(SizeInBytes & 0xFFFFFFFF), (int) (SizeInBytes >> 32));
```
Comment (olkononenko) [inline Public/Src/Engine/Dll/HistoricTableSizes.cs]: this is dirty hacking - i'd appreciate any input here

Comment (dannyvv) [reply Public/Src/Engine/Dll/HistoricTableSizes.cs]: Another alternavie would be using bitconverter to create a bytearray to be add to hascode helper. But I like this pattern better way more effecient.

Allthough perhaps this belongs inside the hasCodehelper itself.


---
In reply to: [291774166](https://github.com/microsoft/BuildXL/pull/432#discussion_r291774166) [](ancestors = 291774166)

**Comment by dannyvv** [comment]
Note: The title has a Typo, I corrected it in the PR, but GitHub doesn't pick that up when merging so you'll have to manually do it again on merge....

**Pull request merged**